subsys/bluetooth/host/scan.c

```diff
@@ -1794,6 +1838,8 @@ static int bt_le_per_adv_sync_create_cancel(
 		return -EINVAL;
 	}
 
+	bt_le_scan_update_and_reconfigure(SCAN_ENABLED_REASON_SYNC_SYNCING, BT_HCI_LE_SCAN_DISABLE);
```

Comment:
Ditto: Why this scan reason when we disable scan?

Reply:
Will try to make this clearer in the documentation for bt_le_scan_update_and_reconfigure().